frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureReleasePlanCard/FeatureReleasePlanCard.tsx

```diff
@@ -47,37 +68,88 @@ const StyledTopRow = styled('div')(({ theme }) => ({
     width: '100%',
 }));
 
+const UseButton = styled(Button)(({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+    '&:hover': {
```

Comment:
If you're styling these yourself, make sure that `focus-visible` is also getting the appropriate styling (for tab focus)